.github/workflows/assets.yml

```diff
@@ -117,7 +119,7 @@ jobs:
             # Add ARCH prefix
             new_name="${ARCH}.${base_name}"
             # Rename the file
-            mv "$asset" "assets/$new_name"
+            mv "$file" "assets/$new_name"
```

Comment:
Is the same should be done here?
https://github.com/lf-edge/eve/blob/d38d00782bc4e151704771e6f466e1673996ac39/.github/workflows/assets.yml#L118

Reply:
Yes, fixed!